apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/ConfigServerEurekaServerConfigure.java

```diff
@@ -29,4 +36,37 @@
 @EnableEurekaServer
 @ConditionalOnProperty(name = "apollo.eureka.server.enabled", havingValue = "true", matchIfMissing = true)
 public class ConfigServerEurekaServerConfigure {
+
+  @Order(99)
+  @Configuration
+  @ConditionalOnProperty(name = "apollo.eureka.server.security.enabled", havingValue = "true", matchIfMissing = false)
+  static class EurekaServerSecurityConfigurer extends WebSecurityConfigurerAdapter {
+
+    private static final String EUREKA_ROLE = "EUREKA";
+
+    @Value("${apollo.eureka.server.security.username}")
+    private String username;
+    @Value("${apollo.eureka.server.security.password}")
+    private String password;
```

Comment:
How about let user can config eureka's username and password in configdb?

* only config once even when scale configservice

but the password will save in configdb

Reply:
This is a good idea! Storing the username/password in the DB makes it consistent among multiple config services.
The only issue here is `apollo.eureka.server.security.enabled`, it's hard to read it from DB as it is used in `ConditionalOnProperty`, which is triggered before the database initialization phase.

Reply:
@Anilople I changed the implementation a little and the `apollo.eureka.server.security` related items could be configured in the configdb, please help to take a look.

Reply:
How it work?...

Are there some methods read data the from configdb?...

Reply:
The `BizConfig` would put all the `ServerConfig` properties to the Spring environment. It won't work for `ConditionalOnProperty` as it's initiated later than the `ConditionalOnProperty` processing so I put the `if (eurekaSecurityEnabled)` check in the method.

Reply:
It works.

When I change ServerConfig in configdb

![image](https://user-images.githubusercontent.com/15523186/207051589-aa804a51-a839-4f12-a22c-007f5eb089f4.png)

then try to access http://localhost:8080/eureka/apps/APOLLO-CONFIGSERVICE

it need auth as expect.